### ISSUE TITLE
Breaking change in PasswordState v9.5 for 10 additional fields

### ISSUE THREAD

**Issue by trir262** [dnewsholme/PasswordState-Management#159]
PasswordState v9.5 and onwards present several additional `WebGenericFieldX_ID` fields in the Password payload, which breaks the modules functionality.

Example payload:

```json
{
"PasswordID"           : 11726,
"Title"                : "example account",
"Domain"               : "",
"HostName"             : "",
"UserName"             : "username",
"Description"          : "description",
"GenericField1"        : "",
"GenericField2"        : "",
"GenericField3"        : "",
"GenericField4"        : "",
"GenericField5"        : "",
"GenericField6"        : "",
"GenericField7"        : "",
"GenericField8"        : "",
"GenericField9"        : "",
"GenericField10"       : "",
"GenericFieldInfo"     : {},
"AccountTypeID"        : 12,
"Notes"                : "",
"URL"                  : "https://www.google.com",
"Password"             : "Password",
"ExpiryDate"           : "",
"AllowExport"          : false,
"AccountType"          : "Web Site",
"OTP"                  : "",
"OTPUri"               : "",
"WebUser_ID"           : "",
"WebPassword_ID"       : "",
"WebOTP_ID"            : "",
"WebGenericField1_ID"  : "",
"WebGenericField2_ID"  : "",
"WebGenericField3_ID"  : "",
"WebGenericField4_ID"  : "",
"WebGenericField5_ID"  : "",
"WebGenericField6_ID"  : "",
"WebGenericField7_ID"  : "",
"WebGenericField8_ID"  : "",
"WebGenericField9_ID"  : "",
"WebGenericField10_ID" : "",
}
```

The error message returned in 4.4.46 is 
```text 
Cannot convert value "@{json} value of type "System.Management.Automation.PSCustomObject" to type "PasswordResult"."
At line:190 char:21
+                     [PasswordResult]$PWSEntry = $PWSEntry
+                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : MetadataError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : RuntimeException
```

**Comment by dnewsholme** [comment]
A fix for this was added in PR #158.

Do you have any more details on how to reproduce. I'm using 4.4.46 of the module with passwordstate server build 9.8 (build 9811) and cannot reproduce this error.


> PS C:\Users\Administrator> Get-Module passwordstate-management

ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Script     4.4.46     passwordstate-management            {Add-PasswordStateADSecurityGroup, Copy-PasswordStatePassw...


PS C:\Users\Administrator> Get-PasswordStatePassword


PasswordID  : 1
Title       : testpwd
Username    : testpwd
Password    : EncryptedPassword
Description :
Domain      :

**Comment by trir262** [comment]
Apologies, a simple update-module fixed the issue indeed... 4.4.46 works as expected indeed
